index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a verifiable digital credential (VDC).
+    </p>
+    <p>
+      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, this document includes a registry of credential presentation exchange protocols to ensure interoperability.
+    </p>
+    <p>
+      The API is designed to support the following key properties:
+      <ul>
+        <li>
+          Keep the act of requesting separate from the specific protocol, thereby enabling flexibility and adaptability in both the protocol and credential formats. This way, the pace of change in browsers won't hinder progress nor block new developments.
```

Comment:
This is not guaranteed and is presumptuous that "browsers ... hinder progress" (which is not ideal).  I prefer not to repeat the discussion of the role of user agent, keeping  users safe, IPC restrictions, etc. 

> " This way, the pace of change in browsers won't hinder progress nor block new developments." 

```suggestion
          Keep the act of requesting separate from the specific protocol, thereby enabling flexibility and adaptability in both the protocol and credential formats. 
```

Also, this is wrong with the respect to the rest of the platform... just because this bit could be seen to move faster (it won't), the rest of the platform still adheres to the development that _quote_ "hinder progress" _unquote_. 

This API is even affected right now... for example, AbortController handling is not interoperable right now.

Reply:
I think removing this would be OK ... I think, perhaps, a nicer (and more accepted.) way to put this is in terms of extensibility ... let me try to rephrase this in a suggestion:

```suggestion
          Keep the act of requesting separate from the specific protocol, thereby enabling the extensibility of the protocol and credential formats to be done outside of the browser.
```

Reply:
Suggestion sounds good.

Reply:
I'm still not loving the "the pace of change in browsers won't hinder progress" being there.

Reply:
I think it should just be: 

```suggestion
          Keep the act of requesting separate from the specific protocol, thereby enabling the extensibility of the protocol and credential formats. 
```

Reply:
That works for me